index.js

```diff
@@ -14,10 +15,22 @@ mongoose.connect('mongodb://localhost/foodControl', {
 
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({extended: true}))
+app.use(passport.initialize())
```

Comment:
you are using your own auth provider, you dont need passport anymore

Reply:
Its correct, thanks!